### PR TITLE
Migrate Data article

### PR DESCRIPTION
**Pull request by stefaniehenke**

### CODE REVIEW

File: _docs/articles/tutorials/migrate-data.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: article
+title: Migrate Data to Subscription Accounts
+---
+
+You can migrate project and run data from existing private accounts to workgroups in Professional or Enterprise subscriptions.  
+
+##Migrate Data to a Professional Account
+To migrate data to or from a Professional workgroup, use [Transfer Ownership](/articles/tutorials/transfer-ownership) to transfer ownership of the data to a designated user in the workgroup.
```
Comment (cdye) [inline _docs/articles/tutorials/migrate-data.md]: This sentence feels redundant.. I think the link itself does the job of communicating and can remove the 2nd. So perhaps "...workgroup, [link]Transfer Ownership[link] of the data to a designated user in the workgroup."

File: _docs/articles/tutorials/migrate-data.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+##Migrate Data to a Professional Account
+To migrate data to or from a Professional workgroup, use [Transfer Ownership](/articles/tutorials/transfer-ownership) to transfer ownership of the data to a designated user in the workgroup. 
+
+##Migrate Data to an Enterprise Account
+If you are transferring data to or from an Enterprise workgroup, both the transferring owner and the receiving owner must be members of the workgroup. 
+
+Use the following steps to enable data transfer to an Enterprise account:
+
+1. Create a workgroup and select **Enable collaborators outside this domain** during setup. This option is required to enable data transfers to or from users outside the domain, and cannot be changed after the workgroup is created. See [Manage Workgroups](/articles/tutorials/manage-workgroups) for more information.
```
Comment (cdye) [inline _docs/articles/tutorials/migrate-data.md]: Suggestion: "and select the **Enable collaborators outside this domain** option during setup."

File: _docs/articles/tutorials/migrate-data.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+2. Invite domain members to the workgroup. To simplify transfer management, designate a single user account to receive and accept data ownership requests.
```
Comment (cdye) [inline _docs/articles/tutorials/migrate-data.md]: Suggestion: "Invite members to the workgroup (the members must belong to the domain)."

File: _docs/articles/tutorials/migrate-data.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+3. Invite the data owner to the workgroup. After they have accepted the workgroup invitation, they can follow the instructions in [Transfer Ownership](/articles/tutorials/transfer-ownership) to transfer the data to a designated user in the workgroup.
```
Comment (cdye) [inline _docs/articles/tutorials/migrate-data.md]: Suggestion: "...invitation, they can [link]Transfer Ownership[/link] to a designated user in the workgroup.

**Comment by stefaniehenke** [comment]
I updated the file with the suggested edits. The referenced Workgroups article is now committed as well.

**Comment by ryanwentzel** [comment]
Some of these changes are using invalid syntax. For example, in workgroups-overview.md is using `end callout`. It should be `endcallout`. This is causing problems for the Ops team with the new instances of the app they're attempting to deploy.

**Pull request merged**